spec/rubocop/cop/layout/empty_lines_around_attribute_accessor_spec.rb

```diff
@@ -33,6 +33,66 @@ def do_something
     RUBY
   end
 
+  it 'registers an offense and corrects for a attribute accessor and comment line' do
+    expect_offense(<<~RUBY)
+      attr_accessor :foo
+      ^^^^^^^^^^^^^^^^^^ Add an empty line after attribute accessor.
+      # comment
+      def do_something
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      attr_accessor :foo
+      # comment
+
+      def do_something
+      end
```

Comment:
The behavior that users expect is as follows right?

```suggestion
      attr_accessor :foo
      
      # comment
      def do_something
      end
```

On the other hand, I think the following is expected when `# rubocop: enable`.

```ruby
    # rubocop:disable Style/AccessorGrouping
    attr_accessor :var      # name of global variable to assign it to if any
    attr_accessor :points   # array of [lat, long]
    attr_accessor :color    # css-format color string
    attr_accessor :weight   # width of line in pixels
    attr_accessor :opacity  # from 0.0 to 1.0
    # rubocop:enable Style/AccessorGrouping

    def do_something
```

Furthermore, a corner case is as follows.

```ruby
    # rubocop:disable Style/AccessorGrouping
    attr_accessor :var      # name of global variable to assign it to if any
    attr_accessor :points   # array of [lat, long]
    attr_accessor :color    # css-format color string
    attr_accessor :weight   # width of line in pixels
    attr_accessor :opacity  # from 0.0 to 1.0
    # rubocop:enable Style/AccessorGrouping

    # comment
    def do_something
```

Therefore expected behaviors of regular comment and `#rubocop: enable` comment will be different.

Reply:
Oh, I see... I was occupied thinking in the case of `#rubocop: enable comment`.
Certainly need to adjust the line break position to see if it is a normal comment or a `#rubocop: enable comment` (which is a corner case).
Thank you very much.